[TT-7971, DX-773] Update docs with new 'tyk version' information (#3468)
TykTechnologies/tyk-docs: tyk-docs/content/frequently-asked-questions/check-current-gateway-version.md

weight: 0 
---

> Since gateway version `5.0.8` or `5.2.3` you can inspect detailed build

Reviewer: ```suggestion
Since gateway version `5.0.8` or `5.2.3` you can inspect detailed build
information, including the release version, by running `tyk version`.
```

> Since gateway version `5.0.8` or `5.2.3` you can inspect detailed build

Reviewer: ```suggestion
Since Gateway version `5.0.8` or `5.2.3` you can inspect detailed build
```

information including the release version by running `tyk version`.

> ```

Reviewer: ```suggestion
```console
```

Release version: v5.3.0-dev
Built by:        goreleaser
Build date:      <date>
Commit:          <commit-hash>
Go version:      go1.20
OS/Arch:         linux/amd64
```

> If you need this in a machine readable format, a `--json` flag is available.

Reviewer: Add an example of JSON output

> For older versions of gateway, you can run `tyk --version` to print the

Reviewer: ```suggestion
For older versions of Gateway, you can run `tyk --version` to print the
```

release version for your tyk binary.

The binary is installed in `/opt/tyk-gateway/tyk` by default. If your
binary is not available in your `PATH` environment, invoke it from there.

> You can also see your installed version from the gateway logs. When the

Reviewer: ```suggestion
You can also see your installed version from the Gateway logs during startup.
```

> Gateway starts, it prints the version information into the log output.

Reviewer: remove this sentence, see above shortened version
